src/features/common/Layout/Navbar/getNavBarIcon.tsx

```diff
@@ -26,9 +34,15 @@ function GetNavBarIcon({ mainKey, router, item }: Props): ReactElement {
         }`}
       >
         {router.pathname === item.onclick ? (
-          <GlobeSelected color={themeProperties.primaryColor} />
-        ) : (
+          tenantName === 'salesforce' ? (
+            <GlobeSelected color={themeProperties.primaryColor} />
+          ) : (
+            <HomeLogoSelected color={themeProperties.primaryColor} />
+          )
+        ) : tenantName === 'salesforce' ? (
           <Globe color={themeProperties.light.primaryFontColor} />
+        ) : (
+          <HomeLogo />
```

Comment:
Nested ternary conditions are difficult to read later on. Can this be simplified (perhaps extract some of this logic into a separate function).